airbyte-integrations/connectors/source-news-api/source_news_api/manifest.yaml

```diff
@@ -71,3 +139,191 @@ check:
   stream_names:
     - "everything"
     - "top_headlines"
+
+spec:
+  type: Spec
+  documentation_url: https://docs.airbyte.com/integrations/sources/news-api
+  connection_specification:
+    $schema: http://json-schema.org/draft-07/schema#
+    title: News Api Spec
+    type: object
+    required:
+      - api_key
+      - country
+      - category
+      - sort_by
+    additionalProperties: true
+    properties:
+      api_key:
+        type: string
+        description: API Key
+        airbyte_secret: true
+        order: 0
+      search_query:
+        type: string
+        description: |
+          Search query. See https://newsapi.org/docs/endpoints/everything for 
+          information.
+        examples:
+          - "+bitcoin OR +crypto"
+          - "sunak AND (truss OR johnson)"
+        order: 1
+      search_in:
+        type: array
+        description: |
+          Where to apply search query. Possible values are: title, description,
+          content.
+        items:
+          type: string
+          enum:
+            - title
+            - description
+            - content
+        order: 2
+      sources:
+        type: array
+        description: |
+          Identifiers (maximum 20) for the news sources or blogs you want
+          headlines from. Use the `/sources` endpoint to locate these
+          programmatically or look at the sources index:
+          https://newsapi.com/sources. Will override both country and category.
+        items:
+          type: string
+        order: 3
+      domains:
+        type: array
+        description: |
+          A comma-seperated string of domains (eg bbc.co.uk, techcrunch.com,
+          engadget.com) to restrict the search to.
+        items:
+          type: string
+        order: 4
+      exclude_domains:
+        type: array
+        description: |
+          A comma-seperated string of domains (eg bbc.co.uk, techcrunch.com,
+          engadget.com) to remove from the results.
+        items:
+          type: string
+        order: 5
+      start_date:
+        type: string
+        description: |
+          A date and optional time for the oldest article allowed. This should
+          be in ISO 8601 format (e.g. 2021-01-01 or 2021-01-01T12:00:00).
```

Comment:
@flash1293 can we change this line since it's causing permissions error on Cloud and we plan on using this connector as an example? I think we'd just need to remove the "or"